### PR TITLE
Icon + Viewbox re-work

### PR DESCRIPTION
**Pull request by jsouth**
This was a little more sketchy than anticipated - but the icons are now in a stable state. We will be fine tuning these a bit more.

nlpzfgodt4

Next steps, when appropriate we need to test these changes within the product. 🌟

### CODE REVIEW

**Comment by jondavidjohn** [comment]
One small issue, not sure if this was missed (as it is pretty subtle), but the "waiting" icon might need to expand it's vertical viewbox to account for the jumping dots.

waiting

**Comment by jondavidjohn** [comment]
For example in codeship/shipyard

waiting

**Comment by jsouth** [comment]
Good catch - It's actually not animating for me at all anymore 🤕

**Comment by jsouth** [comment]
**update:** Still experiencing some weird activity: Its rendering better in Firefox, but not in Chrome or other webkit browsers.  🤦‍♂️  

Will not rest until this icon is beautiful again! updates to come!

File: _assets/shipyard/icons/arrow-up.svg [hunk 1]
```diff
@@ -1,3 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 6">
-  <polyline vector-effect="non-scaling-stroke" points="9 9 5 5 1 9" />
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 -1 8 6">
+  <polyline  points="8 4 4 0 0 4"></polyline>
```
Comment (ryanjwilke) [inline _assets/shipyard/icons/arrow-up.svg]: @jsouth Any idea why there's two of these `polyline` tags?

Comment (jsouth) [reply _assets/shipyard/icons/arrow-up.svg]: Really tried to close that one!  Will fix!

**Comment by jsouth** [comment]
@jondavidjohn Per our conversation, adding what we have now and will come back in for additional cleanup next week ⭐️

**Pull request merged**